internal/db/db.go

```diff
@@ -126,6 +127,16 @@ func (d *Database) Close() error {
 	return nil
 }
 
+// EnableLogs enables gorm logging.
+func (d *Database) EnableLogs() {
```

Comment:
Should we reserve this to only tests?

Reply:
we can even remove this and use the env var when logs are needed

Reply:
Happy with that